scripts/testing/programs/match_template/test_template_reproducibility.py

```diff
@@ -62,6 +62,18 @@ def main():
         # Parse command-line arguments
         args = tmArgs.parse_TM_args(wanted_binary_name)
 
+        # Print non-default configuration parameters
```

Comment:
This is not what I intended, please remove this block. I will add a comment in the file where things should have been changed.